specs/integrations/synthetics.md

```diff
@@ -0,0 +1,119 @@
+## Synthetics Integration
+
+Synthetic monitors play a crucial role in periodically checking the status of your services and applications on a global scale. General documentation about synthetic monitors can be found in
+[Synthetics getting started page](https://www.elastic.co/guide/en/observability/current/synthetics-get-started.html).
+
+This integration goes in to more detail about how the sythetics monitors would
+be correlated with the APM traces. Synthetics traces can be categorized in to two
+main types
+  1. HTTP checks - These have one-one mapping with APM transactions
+  2. Browser checks - These have a one-to-many mapping with APM transactions
+
+### Correlation
+
+The Synthetics agent (including Heartbeat) takes the responsibility of creating the
+[`traceparent`](../agents/tracing-distributed-tracing.md#trace_id-parent_id-and-traceparent)
+header for each outgoing network request associated with a test during every
+monitor execution.
+
+- `trace.id` and `parent.id`
+  - outgoing requests that are being explicity traced by the synthetics agent
+     will have the `parent.id` and `trace.id` as part of the trace context.
+  - must be unique for each step for a browser monitor
+  - must be unique for a http monitor
+- `sampled` Flag
+  - used to control the sampling decision for all the downstream services.
+  - 100% sampling when tracing is enabled
+
+#### Browser checks
+
+When executing a Synthetics journey with APM tracing enabled for specific URLs
+using the --apm_tracing_urls flag, the Synthetics agent takes the following
+actions:
+
+1. Adds the traceparent header to each matching outgoing request.
+2. Includes trace.id and parent.id in all the Step Elasticsearch (ES) documents for the journey.
+
+```ts
+// run journey
+npx @elastic/synthetics --apm_tracing_urls "elastic.co/*" 
+
+// example.journey.ts
+journey("elastic e2e", ({ page }) => {
+  step("home page", async() => {
+    await page.goto("https://www.elastic.co")
+  })
+  step("blog page", async() => {
+    await page.goto("https://www.elastic.co/blog")
+  })
+})
+```
+
+Example of the tracing information added to the ES documents for two steps in the journey:
+
+```json
+// Step - homepage
+{"type":"step/end","journey":{"name":"elastic e2e"},"step":{"name":"home page","index":1,"status":"failed","duration":{"us":17382122}}, "trace.id": "xxx"}
+{"type":"journey/network_info","journey":{"name":"elastic e2e"},"step":{"name":"home page","index":1},"http":{"request":{"url":"http://www.elastic.co/","method":"GET"}},"trace.id": "t1", "transaction.id": "tr1"}
```

Comment:
We should use span.id instead of transaction.id here, as the outgoing http call is the equivalent of a span.

```suggestion
{"type":"journey/network_info","journey":{"name":"elastic e2e"},"step":{"name":"home page","index":1},"http":{"request":{"url":"http://www.elastic.co/","method":"GET"}},"trace.id": "t1", "span.id": "s1"}
```